drivers/dma/dma_stm32.c

```diff
@@ -227,10 +223,9 @@ static int dma_stm32_get_periph_increment(enum dma_addr_adj increment,
 DMA_STM32_EXPORT_DMAMUX int dma_stm32_configure(struct device *dev, uint32_t id,
 						struct dma_config *config)
 {
-	struct dma_stm32_data *data = dev->data;
-	struct dma_stm32_stream *stream = &data->streams[id - STREAM_OFFSET];
-	const struct dma_stm32_config *dev_config =
-					dev->config;
+	const struct dma_stm32_config *dev_config = dev->config;
```

Comment:
`config` name would be better to keep code namings consistent.

Reply:
There is/was already a `config` function argument, and I didn't want to change that.

Reply:
Sorry, missed that!